test/remote/gateways/remote_redsys_sha256_test.rb

```diff
@@ -133,6 +136,111 @@ def test_successful_3ds_authorize_with_exemption
     assert_equal 'CardConfiguration', response.message
   end
 
+  def test_successful_3ds_purchase_with_exemption
```

Comment:
Great remote test coverage 🎉 remote suite passes for me with 2 exceptions (same as master)